### PR TITLE
simple static content components

### LINKED ISSUE #832: Create rest of content components (simple sub-components of static pages)
Relies on `Ul`, `Ol`, `BlueCalloutBox` components being built first

Create pages made from pre-existing components (e.g. headings, paragraphs, links, grid-styled components for layout) + simple components mentioned above

- Author Guide
  - [x] [Process](https://submit.elifesciences.org/html/elife_author_instructions.html#process)
  - [x] [Types](https://submit.elifesciences.org/html/elife_author_instructions.html#types)
  - [x] [Full Submission](https://submit.elifesciences.org/html/elife_author_instructions.html#full)
  - [x] [Revised Submissions](https://submit.elifesciences.org/html/elife_author_instructions.html#revised)
  - [x] [Post Decision](https://submit.elifesciences.org/html/elife_author_instructions.html#post)
  - [x] [Journal Policies](https://submit.elifesciences.org/html/elife_author_instructions.html#policies)
  - [x] [Publication Fees](https://submit.elifesciences.org/html/elife_author_instructions.html#fees)
- Reviewer Guide
  - [x] [The Review Process](https://submit.elifesciences.org/cgi-bin/main.plex?form_type=display_rev_instructions#process)
  - [x] [Reviewing policies](https://submit.elifesciences.org/cgi-bin/main.plex?form_type=display_rev_instructions#confidential)
  - [x] [Writing the Review](https://submit.elifesciences.org/cgi-bin/main.plex?form_type=display_rev_instructions#writing)
- Contact Us
  - [x] [Contact eLife](https://submit.elifesciences.org/html/contact.html#)

### PR DESCRIPTION
**Pull request by jsms90**
#### What does this PR do?
- Creates content components listed in issue #832 - i.e. those under the "Author Guide", "Reviewer Guide" & "Contact Us" headings, which do not require any more sub-components to be built
- Removes underlining from our internal links (`Link` from pubsweet), so they look the same as our external links
- Adds correct font to list items
- Add change margin to padding on all headings (H1, H2, etc) so that they align to the 24px baseline grid (see [typography page in Zeplin](https://app.zeplin.io/project/5bacf042615a8b6e20e84a5f/screen/5bc0cddca55ed15fd3fb3973))
- Add margin-bottom to `Paragraph` & `SmallParagraph` (except when they're the last child), so we have `24px` between elements

Please be aware:
- Internal application links will not lead anywhere until #505 is complete & I anticipate potential issues with the internal page link to the FAQ section - see this issue on `react-router-dom`: https://github.com/ReactTraining/react-router/issues/394

#### Any relevant tickets
fixes #832 
relates #840 

#### How has this been tested?
Visually

### CODE REVIEW

File: app/components/pages/AuthorGuide/Full.js [hunk 1]
```diff
@@ -0,0 +1,434 @@
...
+          upload a completed version of this form (
+          <ExternalLink
+            href="https://submit.elifesciences.org/html/transparent_reporting.pdf"
+            target="_blank"
+          >
+            PDF
+          </ExternalLink>
+          ;{' '}
+          <ExternalLink
+            href="https://submit.elifesciences.org/html/transparent_reporting.docx"
```
Comment (tamlyn) [inline app/components/pages/AuthorGuide/Full.js]: This file should probably be hosted somewhere else.

File: app/components/pages/AuthorGuide/Full.js [hunk 2]
```diff
@@ -0,0 +1,434 @@
...
+      <li>
+        <strong>eLife Transparent Reporting Form: </strong>
+        <Paragraph>
+          To facilitate the interpretation and replication of experiments,
+          authors should, where appropriate, provide detailed information in
+          areas relating to sample-size estimation, replicates, and statistical
+          reporting. At the Full Submission step, authors should be ready to
+          upload a completed version of this form (
+          <ExternalLink
+            href="https://submit.elifesciences.org/html/transparent_reporting.pdf"
```
Comment (tamlyn) [inline app/components/pages/AuthorGuide/Full.js]: This file should probably be hosted somewhere else.

File: app/components/pages/AuthorGuide/JournalPolicies.js [hunk 3]
```diff
@@ -0,0 +1,978 @@
...
+      preregistration would be appropriate, noting if they have done so within
+      their cover letter.
+    </Paragraph>
+
+    <H2>Reporting Standards</H2>
+
+    <Paragraph>
+      To facilitate the interpretation and replication of experiments, authors
+      are required to complete eLife&apos;s{' '}
+      <ExternalLink href="/html/transparent_reporting.pdf">
```
Comment (tamlyn) [inline app/components/pages/AuthorGuide/JournalPolicies.js]: This file should probably be hosted somewhere.

File: app/components/pages/ReviewerGuide/ReviewingPolicies.js [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+import ExternalLink from '../../ui/atoms/ExternalLink'
+import List from '../../ui/atoms/List'
+
+const ReviewingPolicies = props => (
+  <React.Fragment>
+    <H1>Reviewing Policies</H1>
+
+    <Paragraph>
+      We encourage our reviewers to familiarise themselves with{' '}
+      <Link href="reviewer-guide/reviewing-policies" target="_blank">
```
Comment (tamlyn) [inline app/components/pages/ReviewerGuide/ReviewingPolicies.js]: `Link` needs a `to`, not `href`. Also should this start with a `/`?

File: app/components/pages/ReviewerGuide/WritingTheReview.js [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+      to complete our transparent reporting form with information relating to
+      sample-size estimation, replicates, and statistical reporting. The form is
+      available to download and consider alongside the other manuscript files.
+    </Paragraph>
+
+    <Paragraph>
+      eLife is a member of the Committee on Publication Ethics (COPE), supports
+      their principles, and follows their flowcharts for dealing with potential
+      breaches of{' '}
+      <ExternalLink href="author-guide/journal-policies" target="_blank">
```
Comment (tamlyn) [inline app/components/pages/ReviewerGuide/WritingTheReview.js]: Should this link start with a `/`?

File: app/components/pages/AuthorGuide/Types.js [hunk 6]
```diff
@@ -0,0 +1,261 @@
...
+    <Paragraph>
+      Research Advances can be of any length, and any number of main display
+      items, but there may only need to be minimal introductory material.
+      Authors have flexibility in the format, for example with a combined
+      Results and Discussion section, and there is no need for a detailed
+      Materials and Methods section when the methods are the same as the
+      original paper. Authors should upload a completed version of the
+      Transparent Reporting Form (
+      <ExternalLink
+        href="https://submit.elifesciences.org/html/transparent_reporting.pdf"
```
Comment (tamlyn) [inline app/components/pages/AuthorGuide/Types.js]: This file should probably be hosted somewhere else.

File: app/components/pages/AuthorGuide/Types.js [hunk 7]
```diff
@@ -0,0 +1,261 @@
...
+        target="_blank"
+      >
+        PDF
+      </ExternalLink>
+      ;{' '}
+      <ExternalLink
+        href="https://submit.elifesciences.org/html/transparent_reporting.docx"
```
Comment (tamlyn) [inline app/components/pages/AuthorGuide/Types.js]: This file should probably be hosted somewhere else.

File: app/components/pages/AuthorGuide/Full.js [hunk 8]
```diff
@@ -0,0 +1,434 @@
...
+          C++, Java). Any code provided should be properly documented, in line
+          with{' '}
+          <ExternalLink
+            href="http://journals.plos.org/ploscompbiol/s/materials-and-software-sharing#loc-sharing-software"
+            target="_blank"
+          >
+            these instructions
+          </ExternalLink>{' '}
+          (courtesy of PLOS). Please also refer to our{' '}
+          <ExternalLink href="/author-guide/journal-policies">
```
Comment (tamlyn) [inline app/components/pages/AuthorGuide/Full.js]: Should this be an internal link?

File: app/components/pages/AuthorGuide/Revised.js [hunk 9]
```diff
@@ -0,0 +1,154 @@
...
+      figure files prior to upload. If authors intend to select the option to
+      have the accepted version of their article published, it will be useful if
+      the typeset PDF of the article is uploaded as well.
+    </Paragraph>
+
+    <Paragraph>
+      Please also make sure to include information in your manuscript related to
+      the use of cell lines, animal or human experimentation and data and
+      software availability, as indicated in our{' '}
+      <ExternalLink href="/author-guide/journal-policies">
```
Comment (tamlyn) [inline app/components/pages/AuthorGuide/Revised.js]: Should this be an internal link?

File: app/components/pages/AuthorGuide/EditorialProcess.js [hunk 10]
```diff
@@ -0,0 +1,313 @@
...
+    <Paragraph>
+      Yes, providing it has been posted on a recognised public preprint server
+      and has a persistent ID. Preprints should be cited as follows: -
+      Author(s). Year. Title. Name of preprint server. doi or url For example: -
+      Narasimhan VM et al. 2016. A direct multi-generational estimate of the
+      human mutation rate from autozygous segments seen in thousands of
+      parentally related individuals. bioRxiv. doi:
+      https://doi.org/10.1101/059436 - Chung J, Gulcehre C, Cho K, Bengio Y.
+      2014. Empirical evaluation of gated recurrent neural networks on sequence
+      modeling. arXiv. http://arxiv.abs/1412.3555
```
Comment (tamlyn) [inline app/components/pages/AuthorGuide/EditorialProcess.js]: In the original, this paragraph is presented as a list but here it gets flattened into one block of text.

File: app/components/pages/AuthorGuide/EditorialProcess.js [hunk 11]
```diff
@@ -0,0 +1,313 @@
...
+
+    <Paragraph>
+      To help increase the accessibility of research and ensure that it is
+      communicated as rapidly as possible, authors are encouraged to make use of
+      preprint servers, such as{' '}
+      <ExternalLink href="http://biorxiv.org/" target="_blank">
+        bioRxiv
+      </ExternalLink>{' '}
+      or
+      <ExternalLink href="https://www.authorea.com/" target="_blank">
```
Comment (tamlyn) [inline app/components/pages/AuthorGuide/EditorialProcess.js]: Missing space between `or` and link.

File: app/components/pages/AuthorGuide/EditorialProcess.js [hunk 12]
```diff
@@ -0,0 +1,313 @@
...
+        target="_blank"
+      >
+        ASAPbio
+      </ExternalLink>{' '}
+      website. Authors can upload a preprint to{' '}
+      <ExternalLink href="http://biorxiv.org/" target="_blank">
+        bioRxiv
+      </ExternalLink>{' '}
+      or
+      <ExternalLink href="https://www.authorea.com/" target="_blank">
```
Comment (tamlyn) [inline app/components/pages/AuthorGuide/EditorialProcess.js]: Missing space between `or` and link.

File: app/components/pages/AuthorGuide/EditorialProcess.js [hunk 13]
```diff
@@ -0,0 +1,313 @@
...
+        template
+      </ExternalLink>{' '}
+      or{' '}
+      <ExternalLink
+        href="https://www.overleaf.com/latex/templates/elife-latex-template/csqxykvsnyxm"
+        target="_blank"
+      >
+        open it directly in Overleaf
+      </ExternalLink>{' '}
+      .
```
Comment (tamlyn) [inline app/components/pages/AuthorGuide/EditorialProcess.js]: Extra space between link and full stop.

File: app/components/pages/AuthorGuide/Fees.js [hunk 14]
```diff
@@ -0,0 +1,275 @@
...
+    <H1>Publication Fees FAQs</H1>
+
+    <Paragraph>
+      A fee of $2,500* is collected for published papers; however, authors with
+      insufficient funding to pay the fee are eligible for a fee waiver.
+    </Paragraph>
+
+    <Paragraph>
+      The publication fee (also referred to as an “open-access” or
+      &quot;article-processing&quot; fee) is a charge collected by many
```
Comment (tamlyn) [inline app/components/pages/AuthorGuide/Fees.js]: In the original, both `open-access` and `article-processing` are surrounded by curly quotes but here the latter has straight quotes.

File: app/components/pages/AuthorGuide/Full.js [hunk 15]
```diff
@@ -0,0 +1,434 @@
...
+          career, women, and experts from places other than USA and Europe.
+          Please do not list colleagues who are close associates, collaborators,
+          or family members. Authors may also provide the names of reviewers or
+          editors who they would prefer to exclude from the assessment of the
+          article. We will make every effort to follow author requests for
+          excluded individuals unless the editors judge that such exclusion
+          would interfere with the rigorous assessment of the work.
+        </Paragraph>
+      </li>
+    </List.Ordered>
```
Comment (tamlyn) [inline app/components/pages/AuthorGuide/Full.js]: Missing final paragraph:
> If your Full Submission has been peer reviewed and you have been asked to make revisions, please review our guidelines for Revised Submissions.

File: app/components/pages/AuthorGuide/JournalPolicies.js [hunk 16]
```diff
@@ -0,0 +1,978 @@
...
+        <strong>Research Conducted by eLife.</strong> As a way of improving our
+        services, we periodically undertake research and surveys relating to
+        eLife&apos;s submission and review process. Where appropriate we will
+        share our findings so that others can benefit. Participation does not
+        affect the decision on manuscripts under consideration, or our policies
+        relating to the confidentiality of the review process. If you would like
+        to opt out of eLife&apos;s research and/or surveys, please{' '}
+        <ExternalLink href="mailto:editorial@elifesciences.org" target="_blank">
+          contact the journal office
+        </ExternalLink>
```
Comment (tamlyn) [inline app/components/pages/AuthorGuide/JournalPolicies.js]: 👍

File: app/components/pages/AuthorGuide/Types.js [hunk 17]
```diff
@@ -0,0 +1,261 @@
...
+
+import Paragraph from '../../ui/atoms/Paragraph'
+import CalloutTextBox from '../../ui/atoms/CalloutTextBox'
+import ExternalLink from '../../ui/atoms/ExternalLink'
+
+const Types = props => (
+  <React.Fragment>
+    <H1>Article Types</H1>
+
+    <Paragraph>eLife publishes the following article types</Paragraph>
```
Comment (tamlyn) [inline app/components/pages/AuthorGuide/Types.js]: Missing full stop.

File: app/components/pages/ContactUs/ContactELife.js [hunk 18]
```diff
@@ -0,0 +1,37 @@
...
+      having difficulties. If you have any further questions about submitting,
+      submitted, or accepted articles, please contact our editorial or
+      production teams:
+    </Paragraph>
+
+    <List.Unordered>
+      <li>editorial@elifesciences.org</li>
+      <li>production@elifesciences.org</li>
+      <li>editorial@elifesciences.org</li>
+    </List.Unordered>
```
Comment (tamlyn) [inline app/components/pages/ContactUs/ContactELife.js]: The original page makes some (almost certainly futile) attempt at obfuscating the email addresses. I assume we're OK with no longer doing this?

Comment (jsms90) [reply app/components/pages/ContactUs/ContactELife.js]: Ah. I assumed that this was unintentional & just rendering wrongly. @hdrury1 ?

**Comment by diversemix** [comment]
@jsms90 / @giorgiosironi ... Do you think we should host those static PDF files in S3?

**Comment by giorgiosironi** [comment]
> host those static PDF files in S3?

If they are meant to be updated out-of-band by non-technical people, yes. We can create a separate public bucket.

File: app/components/pages/AuthorGuide/EditorialProcess.js [hunk 19]
```diff
@@ -0,0 +1,313 @@
...
+      For manuscripts that are invited for in-depth peer review, we request more
+      detailed information about the work to support the peer review process,
+      for example names of co-authors, details of major datasets, and ethics
+      statements. Authors are asked to agree to publish their work under the
+      terms of the Creative Commons Attribution license (
+      <ExternalLink href="/html/CCBY_4.0.pdf" target="_blank">
+        PDF of the agreement
+      </ExternalLink>
+      ), or the Creative Commons CC0 public domain dedication (
+      <ExternalLink href="/html/CC0_1.0.pdf" target="_blank">
```
[the remark this answers is not in the document]
Comment (jsms90) [reply app/components/pages/AuthorGuide/EditorialProcess.js]: I'll wait until we've got the new hosted URL :+1:

**Comment by giorgiosironi** [comment]
Example of how that would look like:
```
https://continuumtest-elife-figure-viewer.s3.amazonaws.com/index.html
```
You can't really put CNAMEs on top of it without a CDN however.

**Comment by jsms90** [comment]
@diversemix shall we wait until we've got these PDFs hosted before merging?

All done with the rest of the changes, thanks @tamlyn :grin:

**Comment by diversemix** [comment]
@jsms90  let's merge after you raise the ticket and reference it here for the S3 work.

File: app/components/pages/AuthorGuide/EditorialProcess.js (hunk 19 above)
[the remark this answers is not in the document]
Comment (jsms90) [reply app/components/pages/AuthorGuide/EditorialProcess.js]: Ok hosting is going in a separate ticket, so I'll update these to link to eJP for now

**Comment by jsms90** [comment]
@diversemix Raised an issue. So there's just @tamlyn's last comment to address. I'm not sure what the right answer is?

Aside from that, ready to merge unless someone else wants to give these components a once over

File: app/components/pages/ContactUs/ContactELife.js (hunk 18 above)
Comment (diversemix) [reply app/components/pages/ContactUs/ContactELife.js]: @jsms90 Let's go with what is currently live, I think that is a safer assumption - We can always 'fix' the attempt at obfuscation later :)

Comment (tamlyn) [reply app/components/pages/ContactUs/ContactELife.js]: The other thing to mention is that these pages are now generated client side which makes things mush harder for the bots (compared to just updating a regex). Also, spam detection is much better these days.

**Pull request merged**